src/pages/home/report/ReportActionsView.js

```diff
@@ -232,6 +234,19 @@ class ReportActionsView extends React.Component {
         updateLastReadActionID(this.props.reportID, maxVisibleSequenceNumber);
     }
 
+    /**
+     * Finds and updates most recent IOU report action number
+     *
+     * @param {Array<{sequenceNumber, actionName}>} reportActions
```

Comment:
Bad doc as it doesn't return an `Array`

Reply:
This is for function `param` not `return`. Am I missing something here?

Reply:
Nope, just my brain on a Friday misread this.